code/splashscreen.py

```diff
@@ -5,10 +5,10 @@
 def build(root: tk.Tk) -> tk.Label:
     # Load the splash screen image
     try:
-        splash_image: ImageTk.PhotoImage = ImageTk.PhotoImage(Image.open("images/splash.jpg"))
+        splash_image: ImageTk.PhotoImage = ImageTk.PhotoImage(Image.open("/Users/rafaelbalassiano/Desktop/LASER_TAG/Project/images/splash.jpg"))
     
     except:
-        splash_image: ImageTk.PhotoImage = ImageTk.PhotoImage(Image.open("../images/splash.jpg"))
```

Comment:
^

Reply:
I think this commit has the auto populate backwards. I have it fixed I will make a new commit with it